addons/website_sale_delivery/views/res_config.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+    <record id="view_website_config_settings" model="ir.ui.view">
```

Comment:
The view should be named ``website_config_settings_view_form``

See this page for the coding guidelines